pandas/core/indexes/datetimes.py

```diff
@@ -1985,23 +1985,20 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
         we perform reverse operation where we remove tize zone & make it
```

Comment:
Can you just leave the type in `Returns` (so, get rid of the "localized" name). Also, a short explanation on what is returned would be useful.

Reply:
@datapythonista 
I'm understand that I have to remove localized from return section & write like this :
Returns
-------
DatetimeIndex

or I should write some explanation on returns section like this:
Returns
 -------
Some Explanation 
I'm confused about it ?

Reply:
and parameter ambiguous : {'infer', 'NaT', default 'raise'}, bool-ndarray . write like this ???

Reply:
For return, first line as you say, then another line with explanation. You can check the first example here:
https://python-sprints.github.io/pandas/guide/pandas_docstring.html#section-4-returns-or-yields.

For parameter, I didn't see a case exactly like this before, but for the convention I'd say something like:

`{'raise', 'infer', 'NaT'} or bool array-like, default 'raise'`

Reply:
is there any change with summary & with example section